tools/busywork/tcl/fabric.tcl

```diff
@@ -357,7 +357,7 @@ proc ::fabric::caLogin {i_peer i_user i_secret} {
     http::cleanup $token
 }
 
-    
```

Comment:
I'm not aware of one either, but I recently started using [Emacs Prelude](http://batsov.com/prelude/) and it loads some funky defaults for certain things.  It's quite plausible the default configuration for Tcl mode has something like that turned on.